static/js/app.js

```diff
@@ -33,12 +33,20 @@ require('../dist/reporting-views');
           templateUrl: '/partials/analytics.html',
           controller: 'AnalyticsCtrl'
         })
+        .when('/analytics/reporting/:form', {
+          templateUrl: '/partials/analytics.html',
+          controller: 'AnalyticsCtrl'
+        })
+        .when('/analytics/reporting/:form/:facility', {
+          templateUrl: '/partials/analytics.html',
+          controller: 'AnalyticsCtrl'
+        })
```

Comment:
Yeah, add a ? at the end, eg: `/analytics/reporting/:form/:facility?`

Source: https://docs.angularjs.org/api/ngRoute/provider/$routeProvider